api-server/src/server/middlewares/error-handlers.js

```diff
@@ -24,6 +24,8 @@ const isDev = process.env.FREECODECAMP_NODE_ENV !== 'production';
 
 export default function prodErrorHandler() {
   // error handling in production.
+  // NOTE: _next is not used, but has to be there or the middleware will be
+  // silently ignored
```

Comment:
That's because the handler is last in the queue. Is there something wrong in executing it `next()`?

Reply:
I think it's inadvisable, because we've already handled the error, logged it and sent a response i.e. done everything we want to do. Anything after that is either unnecessary or going to cause problems.